### ISSUE TITLE
Foundation theme: Redundant category description

### ISSUE THREAD

**Issue by danni-stark** [vanilla/vanilla#10487]
### Details

Using the Foundation theme,  category description appears to be duplicated in the banner as a banner Description.

![Screen Shot 2020-04-22 at 11 59 24 AM](https://user-images.githubusercontent.com/31856281/80013214-8dfaa600-849c-11ea-82a3-564e21dc8c2f.png)

**Another Example**

![Screen Shot 2020-10-29 at 8 35 58 PM](https://user-images.githubusercontent.com/31856281/97646565-61fd4600-1a26-11eb-9ed1-f44ff9ebdef0.png)

**Comment by andrewdodson** [comment]
We just had another report of this occurring but this time with the keystone theme.

**Comment by igraziatto** [comment]
Keystone hide the category name and description on the banner if the search integration is enabled:
![Screen Shot 2020-04-24 at 10.07.52 AM.png](https://images.zenhubusercontent.com/59f246f2a468affe6df84d2b/d02c33cb-6684-4212-a7a8-075ce29a8f9d)

@andrewdodson do they have customize theme?

**Comment by andrewdodson** [comment]
I feel like this has lost steam, but we have a sub-csm client having issues with this now: 

![image](https://user-images.githubusercontent.com/19805402/89907434-afea7680-dbba-11ea-8a0c-d2e34d43f8be.png)


It looks like Red Cap dealt with it somehow but I don't see any pockets or obvious CSS blocking that second site description on their page.  Any advice or would it be possible to get this issue back on the burner?

**Comment by mike-masson** [comment]
And another
https://vanilla.freshdesk.com/a/tickets/82898

**Comment by dannimajdalani** [comment]
Reeve https://github.com/vanilla/support/issues/3082

**Comment by vrjrobitaille** [comment]
Sounds like the solution here is to:
a) Fix our  default template to show title & description only in banner by default
b) Enhance our Global theming options to allow theme editors the ability to enable/disable titles &descriptions in the banner. If disabled in the banner, they would appear after the banner before the content? 

I've got an roadmap issue for improving banner theming options and dashboard settings here - https://github.com/vanilla/roadmap/issues/132 . I want to close this issue out with that Epic. 

I've added a note to the theme settings subtask for the ability  to control where titles & descriptions are used - https://github.com/vanilla/roadmap/issues/133
